app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Start each argument in the `__init__` from a new line including `self`. `) -> None` should be from a new line as well.